src/main/kotlin/no/nav/dagpenger/regel/periode/PeriodeSpesifikasjoner.kt

```diff
@@ -115,7 +129,6 @@ val ordinær104: Spesifikasjon<Fakta> =
 
 val ordinær: Spesifikasjon<Fakta> = ordinær52 eller ordinær104
```

Comment:
```suggestion
val ordinær: Spesifikasjon<Fakta> = (ordinær52 eller ordinær104) og ikke(erIkkeSærRegel)
```

Ser ut til å kunne funke for 👇

Reply:
cool!

Reply:
fikser og rydder

Reply:
mm, fikk ikke det helt til  å funke, da må være en spesifikasjon?